### ISSUE TITLE
Pronoun reversal

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7307]
**Preferred term label:**
Pronoun reversal

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of language impairment in which incorrect pronouns are used; i.e., using ‘I’ for ‘you’ and/or ‘you’ for ‘I’ (e.g., a child saying ‘You want some more milk’ meaning ‘I want some more milk’). Reversal errors involve the person component, such that ‘I’ and ‘you’ become confused. 

PMID: 27103941

**Comment**
These errors are attested in normal toddlers’ spontaneous speech, albeit at varying rates; pronouns typically emerge around 1.5 years of age and reversals are most common before 2.5 years. A higher rate of pronoun reversal may be expected of people with autism spectrum disorder.

**Parent term (use hpo.jax.org/app)**
Behavioral abnormality

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
synonym: pronominal reversal

**Comment by pnrobinson** [comment]
Child of Abnormal communicative behavior

**Comment by pnrobinson** [comment]
Added as HP:4000072
